platforms/core-execution/persistent-cache/build.gradle.kts

```diff
@@ -15,11 +15,15 @@ dependencies {
     api(projects.logging) {
         because("Because GradleVersion temporarily lives in the logging project until 9.0")
     }
+    api(projects.messaging)
```

Comment:
This seems fishy. Why does our caching infrastructure have a dependency on cross-process messaging?

Reply:
because of `DefaultCrossBuildInMemoryCacheFactory` moved there.